themes/bootstrap5/templates/feedback/form.phtml

```diff
@@ -17,9 +17,7 @@
 ?>
 <?php if (!$this->inLightbox): ?><div class="feedback-content"><?php endif; ?>
   <?php if ($title): ?>
-    <?php $this->headTitle($title); ?>
-    <?php $headTag = $this->inLightbox ? 'h2' : 'h1'; ?>
-    <<?=$headTag?>><?=$this->escapeHtml($title)?></<?=$headTag?>>
+    <?=$this->component('page-title', ['title' => $title])?>
```

Comment:
Is it intentional that there will no longer ever be an h1 here?

Reply:
hmm, most of the headers have h2 as their level, but this one is 1, might have to check it out in these situations.